### PR TITLE
Update hf filesystem "get all files" example

### PR DESCRIPTION
**Pull request by SwayStar123**
The example is wrong. With the example an error is received
```ValueError: Invalid pattern: '**' can only be an entire path component```

### CODE REVIEW

**Comment by SwayStar123** [comment]
@hanouticelina Can you review this

**Review by hanouticelina — APPROVED** [review]
Hello @SwayStar123, thanks for the PR! the correct pattern for recursive matching using `HfFileSystem.glob()` is indeed `**/*.csv` :)

**Comment by SwayStar123** [comment]
I cannot merge this, only someone with write access can @hanouticelina

**Pull request merged**
